docs/current_docs/quickstart/292472-arguments.mdx

```diff
@@ -70,7 +70,7 @@ dagger -m github.com/shykes/daggerverse/hello@v0.1.2 call hello --greeting=bonjo
 
 You should see the same output as before, but in giant letters!
 
-```shell
+```
```

Comment:
Quick note that I've removed the `shell` hint here as it was causing the syntax highlighter to automatically colorize the output snippets in odd ways.